app/views/flush.py

```diff
@@ -0,0 +1,74 @@
+from app.globals import get_answer_store, get_metadata, get_questionnaire_store
+from app.submitter.submitter import SubmitterFactory
+from app.submitter.converter import convert_answers
+from app.utilities.schema import get_schema
+
+from app.authentication.flush_permission_denied import FlushPermissionDenied
+from app.authentication.jwt_decoder import JWTDecryptor
+from app.authentication.map_role_to_permissions import map_role_to_permissions
+from app.authentication.no_survey_data_to_flush import NoSurveyDataToFlush
+from app.authentication.no_token_exception import NoTokenException
+from app.authentication.user import User
+from app.authentication.user_id_generator import UserIDGenerator
+
+from app.questionnaire.path_finder import PathFinder
+
+from flask import Blueprint, Response, request, session
+
+flush_blueprint = Blueprint('flush', __name__)
+
+
+@flush_blueprint.route('/flush', methods=['GET'])
+def flush_data():
+
+    if session:
+        session.clear()
```

Comment:
It confuses the application if you are already login in, as it's gone through an authentication process, then you are asking for it to do again. There is no logically reason why you would be completing a survey and wanting to flush at the same time, so a clean session is the best way to go